drop-transfer/src/service.rs

```diff
@@ -85,14 +101,30 @@ impl Service {
 
             ws::server::spawn(state.clone(), logger.clone(), stop.clone(), guard.clone())?;
 
-            manager::resume(&state, &logger, &guard, &stop).await;
-
-            Ok(Self {
+            let service = Self {
                 state,
                 stop,
                 waiter,
                 logger,
-            })
+                tasks: Arc::new(tokio::sync::Mutex::new(HashMap::new())),
+            };
+
+            {
+                let xfers = service.state.transfer_manager.outgoing.lock().await;
+                for xstate in xfers.values() {
+                    service.trigger_peer_outgoing(xstate.xfer.clone()).await;
+                }
+            }
+
+            {
+                let xfers = service.state.transfer_manager.incoming.lock().await;
+
+                for xstate in xfers.values() {
+                    service.trigger_peer_incoming(xstate.xfer.clone()).await;
+                }
+            }
```

Comment:
Im thinking if it makes sense to trigger all all of the incoming and outgoing states on service start, in the situation that someone has a lot of transfers queued up might not be optimal? Could we not just instead rely on the apps to give us this information? We could for example require this peer state JSON in the config on startup to make sure the apps don't forget to give us the initial state

Reply:
That is actually a fair point. I will try to remove it and see how it goes

Reply:
I've added it and surprisingly to me - no testcase broke because of it which is a bonus

Reply:
nice!